components/server/ee/src/billing/billing-mode.ts

```diff
@@ -76,14 +76,18 @@ export class BillingModesImpl implements BillingModes {
 
         // Is Usage Based Billing enabled for this user or not?
         const teams = await this.teamDB.findTeamsByUser(user.id);
-        const isUsageBasedBillingEnabled = await this.configCatClientFactory().getValueAsync(
-            "isUsageBasedBillingEnabled",
-            false,
-            {
-                user,
-                teams,
-            },
-        );
+        var isUsageBasedBillingEnabled = false;
```

Comment:
nits:
 - `let` (TS) instead of `var` (JS)
 - `await` in `if` clauses is potentially dangerous, so we tend to prefer `const isUBPEnabledForTeam = await ...; if (isUBPEnabledForTeam) { ... };`

Reply:
Why is it dangerous?

Reply:
If you forget the `await`, no linter on this planet will warn you, and it will always resolve to `true`.

Reply:
TIL